tests/Unit/PointwiseFunctions/GeneralRelativity/GrTestHelpers.hpp

```diff
@@ -94,6 +98,11 @@ template <size_t SpatialDim, typename DataType>
 tnsr::Abb<DataType, SpatialDim> make_spacetime_christoffel_second_kind(
     const DataType& used_for_size);
 
+template <size_t SpatialDim, typename DataType>
+tnsr::aBcc<DataType, SpatialDim> make_deriv_spacetime_christoffel_second_kind(
+    const DataType& used_for_size);
+
+
```

Comment:
remove

Reply:
Pretty sure I need this in order to test Ricci

Reply:
The blank line?

Reply:
O, OK. I didn't think blank lines mattered, so I thought you were talking about the function itself.

Reply:
The compiler doesn't care, we do :)

Reply:
Fixed.